src/main/java/seedu/address/logic/commands/people/PeopleRemindallCommand.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands.people;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+import javax.mail.MessagingException;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.User;
+import seedu.address.model.reminder.Reminder;
+
+/**
+ * Reminds everyone in the address book, who owe the user money.
+ */
+public class PeopleRemindallCommand extends Command {
```

Comment:
[Suggestion/Question]
Will `PeopleRemindAllCommand` be better? Make the "all" capital.

Reply:
Yup agree